### ISSUE TITLE
Bitcoin Prices

### ISSUE THREAD

**Issue by otruja** [datahq/awesome-data#21]
The project to apply new knowledge (mainly to test the understanding of the order and the format of the files & directories, preparing datapackage.json with and without CLI, etc.)

**Comment by otruja** [comment]
https://www.coindesk.com/price/ - generates .csv, but restricted by the terms of use, quote,

_You may use the Content online and solely for your personal, non-commercial use, and you may download or print a single copy of any portion of the Content for your personal, non-commercial use, provided you do not remove any trademark, copyright or other notice contained in such Content. You may not, for example, republish the Content on any Internet, Intranet or Extranet site or incorporate the Content in any database, compilation, archive or cache or store the Content in electronic form on your computer or mobile device unless otherwise expressly permitted byCoinDesk_

**Comment by otruja** [comment]
Can't find any legal on this [website](https://coinmarketcap.com/currencies/bitcoin).

**Comment by otruja** [comment]
Need advice though on how to workaround data validation error - .csv in archive directory has bad values in the last rows on the column 'Volume". One solution is to request the data only for the dates with valid values, but probably this error might be ignored too (since final data is valid and got no errors).

**Comment by anuveyatsu** [comment]
@Branko-Dj can we get this dataset published?

**Comment by rufuspollock** [comment]
@anuveyatsu what happened here? There's no update and sprint completed.

**Comment by Branko-Dj** [comment]
@rufuspollock @anuveyatsu Data on coinmarketcap.com is protected by copyright https://coinmarketcap.com/terms/. There it says:

> Prohibited Activities
> You agree that you will not:
> 
> Copy, modify or create derivative works of the Service or any Content;
> Copy, manipulate or aggregate any Content (including data) for the purpose of making it available to > > any third party;
> Trade, sell, rent, loan, lease or license any Content or access to the Service, whether commercially or > free of charge;
> Use or introduce to the Service any data mining, crawling, "scraping", robot or similar automated or > > > data gathering or extraction method, or manually access, acquire, monitor or copy any portion of the > > Service, or download or store Content (unless expressly authorized by CMC). Certain data and other > > information within the Service is available by subscription, or for a fee, at https://pro.coinmarketcap.com;

So I scraped cryptocurrency data from a different website coinmetrics.io. Data can be found at https://datahub.io/cryptocurrency

**Comment by zelima** [comment]
@Branko-Dj can we add views to datasets? Nice time series graphs would be great here I think. Once that's done, think we can close this one

**Comment by Branko-Dj** [comment]
@zelima Yes I think that will be nice. I will add some views for better look

**Comment by zelima** [comment]
@Branko-Dj where this dataset lives on GitHub can't find under `datasets` organization https://github.com/datasets/bitcoin-prices. 

Is it still under @otruja , if so @otruja could you please transfer ownership so that I can do modifications to it and re-publish

**Comment by rufuspollock** [comment]
@zelima what is happening here? If necessary just clone the original repo and push directly to github.com/datasets ...

**Comment by Branko-Dj** [comment]
I am currently working on this

**Comment by rufuspollock** [comment]
@Branko-Dj what happened?
